addons/website_blog/static/src/js/wysiwyg.js

```diff
@@ -23,6 +23,13 @@ Wysiwyg.include({
     async start() {
         await this._super(...arguments);
         $('.js_tweet, .js_comment').off('mouseup').trigger('mousedown');
+        if (document.body.querySelector('#o_wblog_post_content')) {
+            document.body.querySelectorAll(["#o_scroll section .container", "#o_scroll section .container-fluid"]).forEach(section => {
+                section.classList.remove("container");
+                section.classList.remove("container-fluid");
+                section.classList.add("o_container_small");
+            });
+        }
```

Comment:
I don't understand that one though, is not that `start` called only once at editor start ? Not on every snippet drop then?

Reply:
It patches once all editor's block template when the editor starts on the blog post page.

Reply:
In which case would that be needed? Default content should just be using the o_container_small class from the start

Reply:
The default content is forced to an `s_text_block` with `o_container_small`.

The code here is about patching the templates of the droppable snippets directly at their source in the menu from where they are drag'n'dropped.